### PR TITLE
Stronger wording and simpler design on the banner

### PR DESCRIPTION
**Pull request by mourner**
Before:

After:

### CODE REVIEW

File: banner2-direct.svg [hunk 1]
```diff
@@ -10,21 +10,6 @@ svg|a:link, svg|a:visited {
...
-svg|a:hover > .button, svg|a:active > .button {
-  fill: white;
-  stroke: #24292f;
-}
-
-svg|a:hover > text, svg|a:active > text {
-  fill: #24292f;
-}
-
 .button {
```
Comment (Tyrrrz) [inline banner2-direct.svg:13]: Can be removed now?

**Review by Tyrrrz — APPROVED** [review]

**Comment by vadimdemedes** [comment]
Something's wrong with the arrow positioning:
CleanShot 2022-04-03 at 21 54 14@2x

Text on top is also slightly too big. Mac, 13 inches.

**Comment by mourner** [comment]
@vadimdemedes damn, seeing this now too in Safari. It looked perfect in Chrome :( Will need adjustments then.

**Comment by mourner** [comment]
@vadimdemedes looks like the truncated right is a problem with Safari SVG embeds through IMG. If you open it directly or embed as object, it works correctly. 🤷‍♂️

**Pull request merged**